package.json

```diff
@@ -38,7 +38,9 @@
     "js-minify": "uglifyjs --compress warnings=false --mangle --comments '/^!/' --output dist/js/bootstrap.min.js dist/js/bootstrap.js",
     "js-minify-docs": "uglifyjs --compress warnings=false --mangle --comments '/^!/' --output docs/assets/js/docs.min.js docs/assets/js/vendor/anchor.min.js docs/assets/js/vendor/clipboard.min.js docs/assets/js/vendor/holder.min.js docs/assets/js/src/application.js",
     "js-test": "phantomjs ./node_modules/qunit-phantomjs-runner/runner.js js/tests/index.html 60",
-    "js-test-cloud": "ruby -run -ehttpd . -p3000 > /dev/null & grunt saucelabs-qunit",
+    "js-test-dep": "npm install grunt && npm install grunt-saucelabs",
```

Comment:
I just want to install these two dependencies that's why, `npm install` will install all the dependencies.
`grunt` isn't a `devDependencies` because we use `grunt` only to trigger cloud test on Saucelabs